crates/ty_python_semantic/resources/mdtest/type_qualifiers/classvar.md

```diff
@@ -64,24 +64,6 @@ c = C()
 c.a = 2
 ```
 
-and similarly here:
-
-```py
-class Base:
-    a: ClassVar[int] = 1
-
-class Derived(Base):
-    if flag():
-        a: int
-
-reveal_type(Derived.a)  # revealed: int
-
-d = Derived()
-
-# error: [invalid-attribute-access]
-d.a = 2
-```
```

Comment:
I'm not sure if it's valuable to keep this test and write a TODO. We do not emit `invalid-attribute-access` here anymore because of the missing union handling (see my other comment). But also, the main behavior is tested above, and the definition of `a: int` on `Derived` should be a inconsistent-overwrite anyway.